### PR TITLE
Wallet Transactions biz logic and some styling

### PR DESCRIPTION
**Pull request by dwalintukan**
**Added functionality for Deposit and Withdraw buttons.**
image

**Deposit Dialog**
image

**Withdraw Dialog**
image

### CODE REVIEW

File: src/components/TransactionSentDialog/index.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        body1: '',
+        body2: '',
+      };
+    }
+
+    return (
+      <Dialog
+        open={Boolean(txReturn)}
+        onClose={this.handleAlertClose}
+        aria-labelledby="alert-dialog-title"
```
Comment (liviavinci) [inline src/components/TransactionSentDialog/index.js]: delete?

File: src/components/TransactionSentDialog/index.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    this.getSuccessText = this.getSuccessText.bind(this);
+    this.getErrorText = this.getErrorText.bind(this);
+  }
+
+  render() {
+    const { classes, txReturn } = this.props;
+
+    let text;
+    if (txReturn) {
+      text = txReturn && txReturn.txid ? this.getSuccessText() : this.getErrorText();
```
Comment (liviavinci) [inline src/components/TransactionSentDialog/index.js]: can we have better names for this block of data?

File: src/components/TransactionSentDialog/index.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    const { intl, txReturn } = this.props;
+    return {
+      title: intl.formatMessage(messages.failureMsg),
+      body1: txReturn.error,
+      body2: '',
+    };
+  }
+
+  handleAlertClose() {
+    window.location.reload();
```
Comment (liviavinci) [inline src/components/TransactionSentDialog/index.js]: we can reload for now but will need a smoothing handling method in the future

File: src/components/TransactionSentDialog/styles.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const styles = (theme) => ({
+  bodyText1: {
```
Comment (liviavinci) [inline src/components/TransactionSentDialog/styles.js]: better naming

File: src/languageProvider/locales/en_US.json [hunk 5]
```diff
@@ -87,26 +87,35 @@
...
   "topbar.event": "Event",
   "topbar.events": "Events",
   "topbar.setting": "Setting",
   "topbar.voting": "Voting",
+  "transactionSentDialog.successMsg": "Success",
+  "transactionSentDialog.failureMsg": "Oops, something went wrong",
+  "transactionSentDialog.waitingMsg": "Transaction sent. Waiting for confirmations.",
   "walletHistory.transactionHistory": "Transaction History",
   "walletHistory.time": "Time",
-  "walletHistory.from": "From",
```
Comment (liviavinci) [inline src/languageProvider/locales/en_US.json]: Can we keep a list of missing translation somewhere?

File: src/components/TransactionSentDialog/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/components/TransactionSentDialog/index.js]: done

File: src/components/TransactionSentDialog/index.js (hunk 2 above)
Comment (dwalintukan) [reply src/components/TransactionSentDialog/index.js]: renamed to `textContent`

File: src/components/TransactionSentDialog/index.js (hunk 3 above)
Comment (dwalintukan) [reply src/components/TransactionSentDialog/index.js]: ya i personally think we should not force reload, but then we should go refreshes on each page this is used. lets do this in a followup PR.

File: src/components/TransactionSentDialog/styles.js (hunk 4 above)
Comment (dwalintukan) [reply src/components/TransactionSentDialog/styles.js]: done

File: src/languageProvider/locales/en_US.json (hunk 5 above)
Comment (dwalintukan) [reply src/languageProvider/locales/en_US.json]: this wasn't deleted, just moved from `walletHistory.from` to `str.from`. we've been using `str.xxx` for generic strings that can be used in more than one place.

**Pull request merged**